### PR TITLE
Orders Update, Delete, Sort

### PR DESCRIPTION
**Pull request by BuneaAlex**

### CODE REVIEW

File: ordersPage.js [hunk 1]
```diff
@@ -79,6 +80,12 @@ export function addOrders(orderData)
     ticketsInput.addEventListener("change",() => {
       orderChangeHandler(ticketCategorySelect,ticketsInput,updateButton);
     });
+
+    deleteButton.addEventListener("click", () => {
+      DeleteOrder(order.orderID).then(data => console.log("deleted",data));
+      ordersContainer.removeChild(orderCard);
```
Comment (karinasigartau0798) [inline ordersPage.js]: You need to call `removeChild` function in then block.

File: src/api_calls/orders_calls.js [hunk 2]
```diff
@@ -51,3 +51,28 @@ export async function getAllOrders()
         });
 }
 
+export function DeleteOrder(id){
```
Comment (karinasigartau0798) [inline src/api_calls/orders_calls.js]: You can rename this function to `deleteOrder`.

File: src/api_calls/orders_calls.js [hunk 3]
```diff
@@ -51,3 +51,28 @@ export async function getAllOrders()
...
+    console.log('inainte de fetch delete')
```
Comment (karinasigartau0798) [inline src/api_calls/orders_calls.js]: Please remove this `console.log`.

File: ordersPage.js (hunk 1 above)
Comment (BuneaAlex) [reply ordersPage.js]: Oops, mi-a scapat. Modific imediat.

File: src/api_calls/orders_calls.js (hunk 2 above)
Comment (BuneaAlex) [reply src/api_calls/orders_calls.js]: ok

File: src/api_calls/orders_calls.js (hunk 3 above)
Comment (BuneaAlex) [reply src/api_calls/orders_calls.js]: ok

File: src/api_calls/orders_calls.js [hunk 4]
```diff
@@ -76,3 +76,28 @@ export function DeleteOrder(id){
 
 }
 
+export function UpdateOrder(id,patchRequestBody){
```
Comment (karinasigartau0798) [inline src/api_calls/orders_calls.js]: You can rename this function to `updateOrder`.

File: ordersPage.js [hunk 5]
```diff
@@ -86,6 +86,25 @@ export function addOrders(orderData)
...
+    updateButton.addEventListener("click", () => {
+      const numberOfTickets = ticketsInput.value;
+      const ticketCategory = ticketCategorySelect.value;
+      const patchRequestBody = {
+        numberOfTickets: numberOfTickets,
+        ticketType: ticketCategory
+      }
+      UpdateOrder(order.orderID,patchRequestBody)
+      .then(data => {
+        console.log("updated",data)
```
Comment (karinasigartau0798) [inline ordersPage.js]: Please remove this `console.log`.

File: src/api_calls/orders_calls.js (hunk 4 above)
Comment (BuneaAlex) [reply src/api_calls/orders_calls.js]: I-am dat remane in commint-ul acesta: 
Small fixes to order calls

File: ordersPage.js (hunk 5 above)
Comment (BuneaAlex) [reply ordersPage.js]: Ok, ii dau imediat.

File: src/api_calls/orders_calls.js [hunk 6]
```diff
@@ -76,3 +76,28 @@ export function DeleteOrder(id){
...
+    console.log('inainte de fetch post'+JSON.stringify(patchRequestBody));
```
Comment (karinasigartau0798) [inline src/api_calls/orders_calls.js]: Please remove this `console.log`.

Comment (BuneaAlex) [reply src/api_calls/orders_calls.js]: Intrebare, sterg toate console.log-urile din tot proiectul?:)) Si alea din catch(error ...)

File: ordersPage.js [hunk 7]
```diff
@@ -86,6 +86,25 @@ export function addOrders(orderData)
...
+        ticketsInput.setAttribute('data-initial-value', data.numberOfTickets);
+        ticketCategorySelect.setAttribute('data-initial-value', data.ticketCategory.description);
+        disableButton(updateButton);
+        var priceParagraph = eventDescription.querySelector('p:nth-child(5)');
```
Comment (karinasigartau0798) [inline ordersPage.js]: You can use `let` instead if `var`.

File: src/api_calls/utils.js [hunk 8]
```diff
@@ -18,3 +18,28 @@ export function authenticationHeaderSetter(headers)
...
+    authenticationHeaderSetter(headers);
+
+    let options = { method: 'GET',
+    headers: headers,
+     mode: 'cors'
+    };
+
+    const url = 'http://localhost:8080/management/logout';
+
+    return await fetch(url,options)
```
Comment (karinasigartau0798) [inline src/api_calls/utils.js:35]: Please remove `await`.

File: src/api_calls/orders_calls.js [hunk 9]
```diff
@@ -50,7 +50,7 @@ export async function getAllOrders()
         });
 }
 
-export function deleteOrder(id){
+export async function deleteOrder(id){
```
Comment (karinasigartau0798) [inline src/api_calls/orders_calls.js]: It is not necessary to declare these function `async` if you do not use `await` in function block.

File: src/api_calls/orders_calls.js [hunk 10]
```diff
@@ -75,7 +75,7 @@ export function deleteOrder(id){
 
 }
 
-export function updateOrder(id,patchRequestBody){
+export async function updateOrder(id,patchRequestBody){
```
Comment (karinasigartau0798) [inline src/api_calls/orders_calls.js]: It is not necessary to declare these function `async` if you do not use `await` in function block.

**Pull request merged**